bootstrap/src/adapter.ts

```diff
@@ -0,0 +1,50 @@
+import { Requester, Validator, AdapterError } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+const inputParams = {
+  endpoint: false,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config, endpoints) => {
```

Comment:
Not sure what this should be named, it's more like a function selector. Somehow execute feels weird to me.

Reply:
Yes, I thought about it as well. Mot sure yet what to do with it, I left it as it is in order to keep same reference across the common adapter interface and endpoint interfaces, until we investigate more.